addons/payment/models/payment_acquirer.py

```diff
@@ -125,6 +125,29 @@ def _valid_field_parameter(self, field, name):
     show_done_msg = fields.Boolean(compute='_compute_view_configuration_fields')
     show_cancel_msg = fields.Boolean(compute='_compute_view_configuration_fields')
 
+    @api.model
+    def _neutralize_fields(self, provider, fields):
+        """ Helper to replace neutralize a acquirer accounts for a specific provider
+            :param str provider: name of provider
+            :param list fields: list of fiels to update to `dummy`
+        """
+        acquirers = self.search([('provider', '=', provider)])
+        neutralized = {field_name: 'dummy' for field_name in fields}
+        acquirers.write(neutralized)
+        for acquirer in acquirers:
+            _logger.info('%s acquirer `%s` account neutralized', acquirer.provider, acquirer.name)
+        not_neutralized_domain = [('provider', '=', provider)] + [(field_name, '!=', 'dummy') for field_name in fields]
```

Comment:
The domain is not correct: it retrieves the providers that have *all* fields different from `'dummy'`.
It should retrieve the providers that have *some* fields different from `'dummy'`...